### DISCUSSION TITLE
Gulp support

### DISCUSSION THREAD

**Discussion by manik-ushur** [vercel/turborepo#3652] [Help] [answered]
Hi Team,

I have a project with multiple sub projects with some of them are in backbone js (built using gulp) and some of them in react (built using craco).

I understand turbo repo can be plugged in with react. Is there an option to plug it in with backbone js with gulp?
I found this package https://www.npmjs.com/package/turbo-gulp. I believe this one is not yours.

**Comment by mehulkar** [comment, marked as answer]
There is nothing preventing you from using Gulp along with Turborepo. You can invoke Gulp from your package.json scripts and Turborepo will not mind. If you run into any specific problems or have a more specific question, please let us know!

**Reply by tmm1** [reply]
One point of pain is that tasks are generally defined in the gulpfile.js and run with `gulp <task>`, so to take advantage of turbo you have to make wrapper scripts for each task in your package.json

If the turbo.json had the ability to run gulp tasks directly that would create a better experience.
